geomstats/geometry/discrete_curves.py

```diff
@@ -356,29 +376,212 @@ def normalize(self, point):
         return gs.einsum("...ij,...->...ij", point, 1 / self.length(point))
 
 
-class SRVTransform(Diffeo):
-    """SRV transform.
+class ClosedDiscreteCurvesStartingAtOrigin(NFoldManifold):
+    r"""Space of discrete closed curves modulo translations.
 
-    Diffeomorphism between discrete curves starting at origin with
-    `k_sampling_points` and landmarks with `k_sampling_points - 1`.
+    Each individual curve is represented by a 2d-array of shape `[
+    k_sampling_points - 1, ambient_dim]`.
+
+    This space corresponds to the space of immersions defined below, i.e. the
+    space of smooth functions from an interval I into the ambient Euclidean
+    space M, with non-vanishing derivative.
+
+    .. math::
+        Imm(I, M)=\{c \in C^{\infty}(I, M) \|c'(s)\|\neq 0 \forall s \in I \},
+
+    where the interval of parameters I is taken to be I = [0, 1]
+    without loss of generality.
 
     Parameters
     ----------
-    ambient_manifold : Manifold
-        Manifold in which curves take values.
+    ambient_dim : int
+        Dimension of the ambient Euclidean space in which curves take values.
     k_sampling_points : int
         Number of sampling points.
+    equip : bool
+        If True, equip space with default metric.
+    """
+
+    def __init__(self, ambient_dim=2, k_sampling_points=10, equip=True):
+        ambient_manifold = Euclidean(ambient_dim)
+        self.closed = True
+        super().__init__(ambient_manifold, k_sampling_points - 1, equip=equip)
+
+        self._quotient_map = {
+            (SRVMetric, "rotations"): (
+                SRVRotationBundle,
+                SRVRotationQuotientMetric,
```

Comment:
This is no longer the right way to define the quotient maps, I believe we no longer have new classes for each quotient metric. See how it is defined for (open) discrete curves.